### ISSUE TITLE
Haxe 3 Haxelib package

### ISSUE THREAD

**Issue by Richard-Walton** [jasononeil/compiletime#2]
Hi Jason,

I'm upgrading my project to be haxe 3 compatible and have found a problem with the CompileTime package (version 2.2.0).

When compiling my project I get the following error:

<code>Unknown identifier : CompileTime</code>

My compiler args definitely have the compile time library in them:

<code>
haxe  -cp src -lib mloader -lib msignal -lib munit -lib mconsole -lib createjs -lib jQueryExtern -lib hscript -lib haxe-ga -lib compiletime -js  path/to/Main.hx
</code>

Looking at other haxelib packages I can see that none have a "src" folder in them.  I

Indeed, if I move the 2 classes contained into "src" into the root of the package folder my application will compile without issue.  Actually, I've just compared the haxe 3 package with the haxe 2 one and noticed that the Haxe 2 one has the classes in the root folder.

In short, I think the haxe 3 package needs updating to put the classes into the correct (root) folder.

Rich.
